remappings.txt

```diff
@@ -1,5 +1,6 @@
 @openzeppelin/contracts/=node_modules/@openzeppelin/contracts/
 @prb/math/=node_modules/@prb/math/
-@sablier/lockup/=node_modules/@sablier/lockup/src/
+@sablier/lockup/=node_modules/@sablier/lockup/
+@sablier/lockup-precompiles=node_modules/@sablier/lockup/precompiles/
```

Comment:
Any reason to bring it back? It was removed in https://github.com/sablier-labs/airdrops/pull/14 because its not used anywhere.

Reply:
No. Let's remove it.

Reply:
https://github.com/sablier-labs/airdrops/pull/17/commits/a39cdd3e699e968003eecfb2f9233d87033eb332